contrib/openshift/README.md

```diff
@@ -63,6 +71,14 @@ Perform the Following steps to access skydive UI
  - Use username = `admin` , Password = `password`  
  - press **sign-in**  
 
+Perform the Following steps to access monitoring data in prometheus
+ - Point your browser to prometheus end-point  
+ `oc get route prometheus -o jsonpath='http://{.spec.host}'`
+ - Go to the popdown menu Status/Targets
+ - Verify that you see the http://skydive-prometheus-connector:9100/metrics endpoint
+ - Go back to the home page
+ - In the query bar, write the variable `skydive_network_connection_total_bytes` and Execute
```

Comment:
@KalmanMeth I chould not see metrics (e.g. skydive_network_connection_total_bytes)  ... looks like something is not workking as you expect:

The logs of the pod looks ok, I see connection ...  but there are no metrics

```2021-03-02T13:33:34.749Z        INFO    core/classify.go:103 NewClassifySubnetFromList  : Cluster net masks: []
2021-03-02T13:33:34.749Z        INFO    pkg/store_prom.go:221 NewStorePrometheus        : prometheus skydive port = 9100
2021-03-02T13:33:34.749Z        INFO    pkg/store_prom.go:226 NewStorePrometheus        : connection timeout = 60
2021-03-02T13:33:34.749Z        INFO    core/subscriber.go:59 NewSubscriber     : Subscribing to ws://skydive-analyzer:8082/ws/subscriber/flow with namespace 'flow'
2021-03-02T13:33:34.750Z        INFO    websocket/client.go:527 (*Client).Connect       : Connecting to ws://skydive-analyzer:8082/ws/subscriber/flow
2021-03-02T13:33:34.767Z        INFO    websocket/client.go:561 (*Client).Connect       : Connected to ws://skydive-analyzer:8082/ws/subscriber/flow
```

Can you re-check ???

Reply:
If there is no capture, then nothing is reported. Verify that we are performing a capture on some valid network interface in your setup.

Reply:
All good now ... I am removing this comment